shopping-cart-service/src/main/java/agh/fis/shopping_cart/controller/ShoppingCartController.java

```diff
@@ -17,17 +21,53 @@ public ShoppingCartController( ProductCatalogClient productCatalogClient) {
         this.productCatalogClient = productCatalogClient;
     }
 
-    @GetMapping("/")
-    public String hello() {
+    @GetMapping("/{id}")
+    public String getCart(@PathVariable String id) {
+        String message;
+        JSONObject json = new JSONObject();
+        json.put("id", 1);
+        json.put("id_customer", 3);
+        json.put("date_created", "20-03-2021");
+
+        JSONArray array = new JSONArray();
+        JSONObject item1 = new JSONObject();
+        item1.put("name", "The Witcher");
+        item1.put("id", 3);
+        item1.put("price", 80.5);
+        item1.put("publisher", "CD Projekt");
+        array.put(item1);
+
+        JSONObject item2 = new JSONObject();
+        item2.put("name", "GTA V");
+        item2.put("id", 4);
+        item2.put("price", 120);
+        item2.put("publisher", "Rockstar Games");
+        array.put(item2);
+
+        json.put("products", array);
+
+        message = json.toString();
+        return message;
```

Comment:
Maybe try to use a data transfer object (ShoppingCartDto?) with fields like ids and a set of items or something like this.  It's not important at this stage to return some real data but to have some structures that will be visible in API. You can try with OpenAPI as I mentioned on Teams